frontend/src/routes/(authenticated)/admin/ProjectTable.svelte

```diff
@@ -53,6 +66,18 @@ async function softDeleteProject(project: Project): Promise<void> {
         notifyWarning($t('delete_project_modal.success', {name: project.name, code: project.code}));
     }
 }
+
+  const migrationStatusTable = {
+    [ProjectMigrationStatus.Migrated]: 'i-mdi-checkbox-marked-circle-outline',
+    [ProjectMigrationStatus.Migrating]: 'loading loading-spinner loading-xs',
+    [ProjectMigrationStatus.Unknown]: 'i-mdi-help-circle-outline',
+    [ProjectMigrationStatus.PrivateRedmine]: 'i-mdi-checkbox-blank-circle-outline',
+    [ProjectMigrationStatus.PublicRedmine]: 'i-mdi-checkbox-blank-circle-outline',
+  } satisfies Record<ProjectMigrationStatus, string>;
+  function migrationStatusIcon(migrationStatus?: ProjectMigrationStatus): string {
```

Comment:
Seems like a minor improvement to me.
```suggestion
  function migrationStatusIcon(migrationStatus?: ProjectMigrationStatus): IconString {
```

Reply:
Sure; now that we've tweaked the IconString definition, this would work.

Reply:
Done in commit https://github.com/sillsdev/languageforge-lexbox/commit/ae57e274b20a511839df463862db09b76406bcc0.